packages/jest-koa-mocks/package.json

```diff
@@ -23,13 +23,13 @@
   },
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/jest-koa-mocks/README.md",
   "dependencies": {
-    "koa": "^2.5.0",
+    "koa": "^2.8.2",
```

Comment:
Bringing our `koa` and `@types/koa` to be more consistent across packages in order to remove duplicate versions.